build.sbt

```diff
@@ -166,19 +173,20 @@ lazy val benchmarks = Project(id = "benchmarks", base = file("benchmarks"))
   .enablePlugins(JmhPlugin)
   .disablePlugins(MimaPlugin)
   .settings(
+    name := s"$pekkoPrefix-benchmarks",
     crossScalaVersions := Dependencies.Versions.CrossScalaForLib,
     scalaVersion := Dependencies.Versions.CrossScalaForLib.head,
     (publish / skip) := true)
   .settings(MetaInfLicenseNoticeCopy.settings)
 
-lazy val docs = Project(id = "pekko-grpc-docs", base = file("docs"))
+lazy val docs = Project(id = "docs", base = file("docs"))
 // Make sure code generation is run:
   .dependsOn(pluginTesterScala)
   .dependsOn(pluginTesterJava)
   .enablePlugins(AkkaParadoxPlugin, ParadoxSitePlugin, PreprocessPlugin, PublishRsyncPlugin)
   .disablePlugins(MimaPlugin)
   .settings(
-    name := "Apache Pekko gRPC",
```

Comment:
While the `name` here is irrelevant, its highly misleading and also quite different to the rest of the pekko projects.